### PR TITLE
Add link to a demo

### PR DESCRIPTION
**Pull request by anssiko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,16 +24,19 @@ How to use
 - Import the motion sensor classes in your HTML (see [`<script type="module">` browser support](https://caniuse.com/#feat=es6-module) or use a [polyfill](https://github.com/ModuleLoader/es-module-loader) to load this polyfill!):
 ```
 <script type="module">
+// Import the classes you need.
```
Comment (kenchris) [inline README.md]: objects?

File: README.md [hunk 2]
```diff
@@ -24,16 +24,19 @@ How to use
...
   Accelerometer,
   LinearAccelerationSensor,
   Gyroscope,
   GravitySensor,
   RelativeOrientationSensor,
   AbsoluteOrientationSensor
 } from './src/motion-sensors.js';
+
+// And they're ready for use!
+sensor = new AbsoluteOrientationSensor();
```
Comment (kenchris) [inline README.md]: You said "you need" but you imported much more than you actually needed :-)

File: README.md [hunk 3]
```diff
@@ -24,16 +24,19 @@ How to use
...
 </script>
 ```
-- That's it. See [`index.html`](https://kenchris.github.io/sensor-polyfills/) ([source](https://github.com/kenchris/sensor-polyfills/blob/master/index.html)) for an example.
+- That's it. See [Orientation phone demo](https://intel.github.io/generic-sensor-demos/orientation-phone/) demo ([code](https://github.com/intel/generic-sensor-demos/blob/master/orientation-phone/index.html)) for an example.
```
Comment (kenchris) [inline README.md]: Is that actually using the polyfills?

File: README.md (hunk 1 above)
Comment (anssiko) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (anssiko) [reply README.md]: Fixed :-)

File: README.md (hunk 3 above)
Comment (anssiko) [reply README.md]: Yes.

**Pull request merged**